target/linux/rockchip/rk3399/base-files/etc/board.d/02_network

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+. /lib/functions/uci-defaults.sh
+
+board_config_update
+
+case "$(board_name)" in
+"pine64,rockpro64")
+	ucidef_set_interface_lan 'eth0'
+	;;
```

Comment:
If it falls into the default case, remove it. If you already have an overview of devices, judge whether the default case is appropriate beforehand.

Reply:
OK :)